dvc/settings.py

```diff
@@ -15,17 +15,17 @@ def __init__(self, msg):
 class Settings(object):
     def __init__(self, argv=None, git=None, config=None):
         self._args = None
-        args = None
+        args is None
```

Comment:
`args = None` should stay in this line since it is an assignment. Could you please revert this line?